packages/app/src/models/Content.tsx

```diff
@@ -0,0 +1,10 @@
+export interface Content {
+  article: string
+  authors: [string]
+  tags: [string]
+  title: string
+  description: string
+  image: string
+  id: string
+  postedOn: number
```

Comment:
@asgeir-eth I created a model file for this since I wanted to pull it in for the PostPreview component I made. The original version of this didn't include the `postedOn` type, and I wasn't sure if adding it here actually made sense or not, since I know it's taken from the block that includes the event.